biz.aQute.junit.platform.reporting/bnd.bnd

```diff
@@ -0,0 +1,8 @@
+# Set javac settings from JDT prefs
+-include: ${workspace}/cnf/includes/jdt.bnd
+
```

Comment:
I am unclear on the purpose of this bundle vs biz.aQute.junit.platform. Perhaps a Bundle-Description describing the role of this bundle would be helpful.

Reply:
See above re: purpose.

I have added `Bundle-Description`.